docs/src/validation_services.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Validation Services
+---
+
+**Subject to change.**
+
+Running a Solana validation-client required relatively modest upfront hardware capital investment. **Table 2** provides an example hardware configuration to support ~1M tx/s with estimated ‘off-the-shelf’ costs:
```

Comment:
I haven't seen anyone benchmark 1,000,000 tps in a long time.  We might want to re-think how we present recommended hardware and it's associated costs/benefits